lib/Elastica/Query/ParentId.php

```diff
@@ -12,29 +12,29 @@ class ParentId extends AbstractQuery
     /**
      * ParentId constructor.
      *
-     * @param string     $type
-     * @param int|string $id
-     * @param bool       $ignoreUnmapped
+     * @param string $type           Name of the child relationship mapped for the join field
+     * @param string $id             ID of the parent document. The query will return child documents of this parent document.
+     * @param bool   $ignoreUnmapped Indicates whether to ignore an unmapped type and not return any documents instead of an error. Defaults to false.
      */
-    public function __construct(string $type, $id, bool $ignoreUnmapped = false)
+    public function __construct(string $type, string $id, bool $ignoreUnmapped = false)
     {
-        $this->setType($type);
+        $this->setRelationshipType($type);
```

Comment:
Haven't check this yet, but I assume this still exists?

Reply:
Don't think so :)

Reply:
this is just a rename of the private method, is "type" here is for the relationship type